app/app/services/sftp_case_transmitter.rb

```diff
@@ -0,0 +1,34 @@
+class SftpCaseTransmitter
+  attr_reader :current_agency, :cbv_flow, :aggregator_report
+
+  def initialize(cbv_flow, current_agency, aggregator_report)
+    @cbv_flow = cbv_flow
+    @current_agency = current_agency
+    @aggregator_report = aggregator_report
+  end
+
+  def deliver_sftp!
+    config = current_agency.transmission_method_configuration
```

Comment:
See above comment regarding the structure of this config hash. This config is referenced below both as strings (`config["sftp_directory"]`) and symbols (`SftpGateway.new(config)`).

```suggestion
    config = current_agency.transmission_method_configuration.with_indifferent_access
```